system.c

```diff
@@ -176,6 +176,14 @@ uint8_t system_execute_line(char *line)
           else { report_ngc_parameters(); }
           break;
 
+        case 'P': // Set force sensitivity parameter. Avoids hastle of changing eeprom settings.
+          char_counter++;
+          if (!read_float(line, &char_counter, &parameter)){
```

Comment:
IIRC, the force sensitivity level is always an integer (or rather a byte: an int between 0 and 255), and indeed it's stored into `settings.force_sensor_level` which is a `uint8_t`. Is `read_float` really the correct function here?